semgrep_runner_v1.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
```

Comment:
Would be nice to have a toplevel comment explaining briefly what is the purpose of this file (you can have a look at semgrep_output_v1.atd for an example, even though it does not have to be that long :) )

Reply:
Maybe a small comment explaining also where is the actual server (I imagine, semgrep-pro/server/...) and the actual client (I imagine somewhere in semgrep-app).

Reply:
Added a comment and included a reference to the server (made this a generic reference to semgrep-proprietary since this is experimental and subject to change)